test/cypress/e2e/moderationEditHealthcareProfessional.cy.ts

```diff
@@ -0,0 +1,90 @@
+import 'cypress-real-events'
+import 'cypress-plugin-tab'
+
+const FAKE_HEALTHCARE_PROFESSIONAL_RESPONSE_PATH = 'moderation_dashboard/fakeModHealthcareprofessionalData.json'
+
+describe('Moderation edit professional healthcare form', () => {
+    context('Landscape mode', () => {
+    // Before starting the execution, we need to load the fixture data using "this" context of the
+    // test object. We need to use regular function() callbacks instead of arrow functions when
+    // we need to access the "this" context.
+        before(function() {
+            cy.fixture(FAKE_HEALTHCARE_PROFESSIONAL_RESPONSE_PATH).then(fakeHealthcareprofessionalResult => {
+                this.fakeHealthcareprofessionalResult = fakeHealthcareprofessionalResult
+            })
+
+            cy.viewport('macbook-16')
+            cy.visit('/login', { timeout: 10000 })
+            Cypress.session.clearCurrentSessionData()
+
+            // This intercepts the call to the GraphQL API in order to use fake data in the tests to protect the real data.
+            cy.intercept('POST', '**/', req => {
+                if (req.body.query && req.body.query.includes('query HealthcareProfessionals')) {
+                    req.reply({
+                        statusCode: 200,
+                        body: this.fakeHealthcareprofessionalResult
+                    })
+                } else {
+                    req.continue()
+                }
+            }).as('getHealthcareProfessionals')
+
+            cy.origin('https://findadoc.jp.auth0.com/', () => {
+                cy.get('input#username').should('be.visible').type('findadoctest@proton.me')
+                cy.get('[data-action-button-primary]').should('be.visible').click()
+                cy.get('input#password').should('be.visible').type('vCnL5J8agHg6m2f')
```

Comment:
this is fine for now as we planned it this way, but normally this would be a big concern to commit a password.

Reply:
(this is just a comment, no need to change it)